apps/frontend/src/modules/workspace/hooks/useWorkspace.tsx

```diff
@@ -58,14 +60,54 @@ export const useWorkspace = () => {
                         },
                     },
                 });
-                return response?.data?.organizationUpdate;
+                if (response?.data?.organizationUpdate) {
```

Comment:
@afolasope the `handleWorkspaceCache` function is already doing the job of this if statement, I thing we call the `handleWorkspaceCache` function with the updated `organization` object.

Reply:
I don't understand this


> @afolasope the `handleWorkspaceCache` function is already doing the job of this if statement, I thing we call the `handleWorkspaceCache` function with the updated `organization` object.

I don't understand this

Reply:
@afolasope What I was saying here was that after calling the `updateOrganization` mutation we could just call `handleWorkspaceCache(updatedOrganizationObject)`, so we don't write the same thing twice.